internal/accounts/create.go

```diff
@@ -56,11 +62,23 @@ var CreateCommand = &command.Command{
 
 func create(
 	_ []string,
-	_ flowkit.ReaderWriter,
+	loader flowkit.ReaderWriter,
 	_ command.GlobalFlags,
 	services *services.Services,
 	state *flowkit.State,
 ) (command.Result, error) {
+	// if user doesn't provide any flags go into interactive mode
+	if len(createFlags.Keys) == 0 {
+		account, err := createInteractive(state, loader)
+		if err != nil {
+			return nil, err
+		}
+		return &AccountResult{
+			Account: account,
+			include: createFlags.Include,
+		}, nil
```

Comment:
@sukantoraymond Here's how I think we can silence the noisy account result.

Rationale: This is an interactive command that already prints to stdout, there's no need to print out the account struct. It takes up a lot of space in the terminal. It's also unlikely that users will consume this command output from another script, so we can remove the JSON output feature.

```suggestion
		err := createInteractive(state, loader)
		if err != nil {
			return nil, err
		}
		return nil, nil
```

- `createInteractive` doesn't need to return an account value (it already prints it itself).
-  We return `nil` rather than returning an `AccountResult` value. This should tell the command framework not to print anything.

However, [this line in command.go](https://github.com/onflow/flow-cli/blob/master/internal/command/command.go#L132-L133) will throw an error if `result` is `nil`.

To avoid that, I think we can just update that function to skip logging results if `result == nil`:

```go
// internal/command/command.go line 129 to 137

handleError("Command Error", err)

if result == nil {
  return
}

// format output result
formattedResult, err := formatResult(result, Flags.Filter, Flags.Format)
handleError("Result", err)

// output result
err = outputResult(formattedResult, Flags.Save, Flags.Format, Flags.Filter)
handleError("Output Error", err)
```